.github/workflows/claim-crate-names.yml

```diff
@@ -4,9 +4,9 @@
 # This workflow claims the names of the unpublished crates in this repository
 # on crates.io (by publishing a dummy empty package)
 
-# Allow only one release to run at a time
+# Allow only one crate claim workflow to run at a time
 concurrency:
-  group: release-smithy-rs
+  group: claim-crates-smithy-rs
```

Comment:
It wouldn't be so good for this workflow to cancel an ongoing release that happens to be running at 11 UTC 😄